SobokSobok/SobokSobok/Presentation/Common/Button/ButtonStyle.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ButtonStyle.swift
+//  SobokSobok
+//
+//  Created by 김승찬 on 2022/02/19.
+//
+
+import UIKit
+
+import SnapKit
+
+enum ButtonMode: Int, CaseIterable {
+    case lightMint
+    case mainMint
+}
+
+final class ButtonStyle: UIButton {
+    
+    var mode: ButtonMode
+    var text: String
```

Comment:
생성자에서 버튼의 스타일이 모두 결정되는 것이고 추후 변경이 되는 것이 아니라면 private하게 만들어주어도 될 것 같아요.
```suggestion
    private var mode: ButtonMode
    private var text: String
```